Libraries/Components/Touchable/TouchableOpacity.js

```diff
@@ -102,20 +109,24 @@ var TouchableOpacity = React.createClass({
    * defined on your component.
    */
   touchableHandleActivePressIn: function() {
-    this.refs[CHILD_REF].setNativeProps({
-      opacity: this.props.activeOpacity
-    });
+    this.clearTimeout(this._hideTimeout);
+    this._hideTimeout = null;
+    this._opacityActive();
     this.props.onPressIn && this.props.onPressIn();
   },
 
   touchableHandleActivePressOut: function() {
-    var child = onlyChild(this.props.children);
-    var childStyle = flattenStyle(child.props.style) || {};
-    this.setOpacityTo(childStyle.opacity === undefined ? 1 : childStyle.opacity);
+    if (!this._hideTimeout) {
+      this._opacityInactive();
+    }
     this.props.onPressOut && this.props.onPressOut();
```

Comment:
yep, delayPressOut is now handled in [Touchable](https://github.com/jmstout/react-native/blob/ad73a3d172b4a1638799d3a27bf375339a3681ff/Libraries/vendor/react_contrib/interactions/Touchable/Touchable.js#L646-L652), per your recommendations.